client/app/admin/create-project/create-project.controller.js

```diff
@@ -92,18 +131,35 @@
          * Create a task grid
          */
         vm.createTaskGrid = function(){
-            // Get the zoom level
-            var zoomLevel = mapService.getOSMMap().getView().getZoom();
+            
+            vm.taskGrid = true;
+            
+            // Remove existing task grid
+            projectService.removeTaskGrid();
 
              // Get the AOI
-            var areaOfInterest = drawService.getFeatures();
+            var areaOfInterest = vm.AOI;
```

Comment:
I was thinking too much ahead (when we'll also have to handle uploaded AOIs). Changed it back to getting it from the service for now.